test/testvm.py

```diff
@@ -698,6 +714,9 @@ def wait_boot(self):
             raise Failure("qemu did not run successfully: %d" % (p.returncode, ))
 
         self.address = address
+        if not self.wait_ssh():
+            raise Failure("Unable to contact machine via ssh.")
```

Comment:
> This will produce weird messages in the machine logs. But I guess that's not a problem.

More meaningful error:
`Unable to reach machine %s via ssh.` where the parameter is the vm's IP address